app-service-deploy/main.js

```diff
@@ -0,0 +1,53 @@
+var minimist = require('minimist');
+var kuduService = require('./azure-arm-app-service-kudu').Kudu;
+
+let mopts = {
+	string: [
+        'action',
+        'scmUri',
+        'username',
+        'password',
+        'packagePath'
+	]
+};
+
+let validActions = ['zipdeploy'];

```

Comment:
should we have constants for each actions as the same action name is being used in switch case ahead.

Reply:
created const